src/League/ColorExtractor/Palette.php

```diff
@@ -10,15 +10,15 @@ class Palette implements \Countable, \IteratorAggregate
     /**
      * @return int
      */
-    public function count()
+    public function count(): int
     {
         return count($this->colors);
     }
 
     /**
      * @return \ArrayIterator
      */
-    public function getIterator()
+    public function getIterator(): \ArrayIterator
```

Comment:
I _think_ this either needs to be `\Traversable` or have a `#[ReturnTypeWillChange]` attribute, but I'm not 100% sure.